app/dashboard/utils.py

```diff
@@ -187,7 +187,8 @@ def get_ipfs(host=None, port=settings.IPFS_API_PORT):
 
     Args:
         host (str): The IPFS host to connect to.
-            Defaults to environment variable: IPFS_HOST.
+            Defaults to environment variable: IPFS_HOST.  The host name should be of the form 'ipfs.infura.io' and not 
```

Comment:
nice add :)

Reply:
This is from the Django shell.  I think I hit all the functions that the app uses (connecting to the infura gateway, retrieving files from a hash, adding/retrieiving json).   Is this what you're thinking?

![ipfs interactions](https://user-images.githubusercontent.com/17355484/59838443-e8772300-931c-11e9-8563-ed0bcb025992.png)

Reply:
I meant more like running the app locally, and then sending around some kudos, creating some test bounties, etc.

Reply:
Oh, sorry, misunderstood.  I hadn't previously but just did.  Sent a couple of kudos and posted 2 new test bounties and seemed to work fine.  

Logs from sending kudos - I don't see any errors (though not 100% sure what I'd be looking for)
> web_1      | 172.19.0.1 - - [22/Jun/2019 10:47:42] "POST /kudos/send/3/ HTTP/1.1" 200 -
web_1      | WARNING 2019-06-22 10:47:45,791 notifications 534 140604666288872 Notifications are disabled.  Skipping email.
web_1      | 172.19.0.1 - - [22/Jun/2019 10:47:45] "POST /kudos/send/4/ HTTP/1.1" 200 -
web_1      | 172.19.0.1 - - [22/Jun/2019 10:47:53] "GET /kudos/marketplace/ HTTP/1.1" 200 -

Logs from posting bounty - I do see that one error but but I believe that's the bounty posting code complaining about me not populating some value in the bounty form - doesn't look like an IPFS issue though
> web_1      | 172.19.0.1 - - [22/Jun/2019 10:43:54] "GET /funding/new HTTP/1.1" 200 -
web_1      | 172.19.0.1 - - [22/Jun/2019 10:43:54] "GET /sync/get_amount?amount=0.001&denomination= HTTP/1.1" 200 -
web_1      | 172.19.0.1 - - [22/Jun/2019 10:43:58] "GET /sync/get_issue_details?url=https%3A%2F%2Fgithub.com%2Facolytec3%2Fnanobitcoin-go%2Fissues%2F2&token=4d36bce0f325b13f672ce9ebabf3c07babe80b6f HTTP/1.1" 200 -
web_1      | 172.19.0.1 - - [22/Jun/2019 10:43:59] "GET /sync/get_issue_details?url=https%3A%2F%2Fgithub.com%2Facolytec3%2Fnanobitcoin-go%2Fissues%2F2&token=4d36bce0f325b13f672ce9ebabf3c07babe80b6f HTTP/1.1" 200 -
web_1      | 172.19.0.1 - - [22/Jun/2019 10:43:59] "GET /api/v0.1/get_suggested_contributors?keywords=&invite= HTTP/1.1" 200 -
web_1      | 172.19.0.1 - - [22/Jun/2019 10:43:59] "GET /api/v0.1/get_suggested_contributors?keywords=&invite= HTTP/1.1" 200 -
**web_1      | ERROR 2019-06-22 10:44:30,310 views 534 140604666288872 expected string or bytes-like object**
web_1      | 172.19.0.1 - - [22/Jun/2019 10:44:30] "GET /funding/details/?url=https://github.com/acolytec3/nanobitcoin-go/issues/2 HTTP/1.1" 200 -
web_1      | 172.19.0.1 - - [22/Jun/2019 10:44:30] "GET /jsi18n/ HTTP/1.1" 200 -
web_1      | 172.19.0.1 - - [22/Jun/2019 10:44:30] "GET /dynamic/js/tokens_dynamic.js HTTP/1.1" 200 -
web_1      | 172.19.0.1 - - [22/Jun/2019 10:44:30] "GET /media/avatars/be43ca7feb8423165e8300b1e6f99877/acolytec3.png HTTP/1.1" 304 -
web_1      | 172.19.0.1 - - [22/Jun/2019 10:44:31] "GET /inbox/notifications/?page=1 HTTP/1.1" 200 -
**web_1      | ERROR 2019-06-22 10:44:31,572 views 534 140604666288872 expected string or bytes-like object**
web_1      | 172.19.0.1 - - [22/Jun/2019 10:44:31] "GET /funding/details/?url=https://github.com/acolytec3/nanobitcoin-go/issues/2 HTTP/1.1" 200 -
web_1      | * confirming tx has mined
web_1      | * getting bounty id
web_1      | ** get_bounty_id_from_web3; looking at 2210
web_1      | * getting bounty
web_1      | * processing bounty

Skipping all the json from the bounties...
>web_1      | - processing changes, None => 6
web_1      | - new_bounty event; diff => None
web_1      | ============ posting ==============
web_1      | ============ done posting ==============
web_1      | changes processed: 
web_1      | {   'did_bsr': False,
web_1      |     'did_post_to_email': False,
web_1      |     'did_post_to_github': False,
web_1      |     'did_post_to_slack': False,
web_1      |     'did_post_to_twitter': False,
web_1      |     'did_post_to_user_discord': False,
web_1      |     'did_post_to_user_slack': False}
web_1      | 172.19.0.1 - - [22/Jun/2019 10:44:51] "POST /sync/web3/ HTTP/1.1" 200 -

Does anything jump out at you as an IPFS related issue?

Reply:
êasiest would be to check if the bounty/kudos/grant has a properly filled ipfs link - if that's the case and it's openable via ipfs.infura.io then it's fine.

Reply:
@danlipert @kuhnchris   Finally found what I was looking for. [This](https://ipfs.infura.io/ipfs/QmaL1ve4XWwt4tU56DKPajHymDACBDHa4W6EVcB7duR11H) is the IPFS hash of a test bounty I created while running this branch locally.  Can we call this one resolved?

Reply:
Awesome @acolytec3 looks great!

Reply:
LGTM

Reply:
Cool, thanks!